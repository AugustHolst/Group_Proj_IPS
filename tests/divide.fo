fun int write_int(int x) = write(x)

fun [int] write_int_array([int] x) = map(write_int, x)

fun [int] main() =
    let a = 5 in
    let b = 5 in
    let d = 2 in
    let zero = 0 in
    let minus = ~8908544 in
    let big = 92302343 in
    let ab = 5/5 in
    let zerod = zero/d in
    let minusa = minus/a in
    let bigminus = big/minus in
    let arr = {ab, zerod, minusa, bigminus} in
    write_int_array(arr)