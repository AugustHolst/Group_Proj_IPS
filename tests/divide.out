1 0 -1781708 -10